### PR TITLE
ember-cli notes 6 July 2017

### PR DESCRIPTION
**Pull request by hjdivad**

### CODE REVIEW

**Comment by locks** [comment]
https://github.com/ember-cli/ember-cli-shims/pull/118

File: ember-cli/2017-07/july-06.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# ember-cli Meeting Notes - 2017-07-06
+
+## Attendees
+
+
+
+## Agenda
```
Comment (kellyselden) [inline ember-cli/2017-07/july-06.md]: release discussion

Comment (kratiahuja) [inline ember-cli/2017-07/july-06.md]: Move : https://github.com/ember-cli/rfcs/pull/108 to FCP (once I update the RFC for same name conflict to default to last one wins with warning only)

**Comment by hjdivad** [comment]
Closing as I don't think anybody took notes.  If someone has something jotted down please feel free to reopen or send it to me and I'll format & put it up

**Pull request closed without merging**